packages/toolpad-app/src/utils/useShortcut.ts

```diff
@@ -16,13 +18,15 @@ export default function useShortcut(
     }
 
     const handleKeydown = (event: KeyboardEvent) => {
-      if (event.code === code && event.metaKey === metaKey) {
+      if (event.code === code && event.metaKey === metaKey && event.shiftKey === shiftKey) {
         handler();
-        event.preventDefault();
+        if (preventDefault) {
```

Comment:
Had to add this as `useShortcut` always prevented default action when I was inside input so default browser undo inside fields did not work.

IMO it could be better to refactor this method by passing event to the handle so that we can handle such cases outside of this function

Reply:
It should be fine to pass the event to the handler, sounds like a better solution to me.

Reply:
I'll keep this out of scope for now as there are quite a few places that would require refactoring to get this working correctly (mostly because of `useCallback` usage, otherwise it should be straight forward)